### DISCUSSION TITLE
Highs Options

### DISCUSSION THREAD

**Discussion by phoenix665** [ERGO-Code/HiGHS#2055] [Q&A] [answered]
Hello,

I am a new user of Highs solver and I am having a bit hard time understanding how to pass options when using Highs on bash Linux. For example if I want to do:

$ highs --log_to_console=false test.lp

I get the following error 

Error parsing options: Option 'log_to_console' does not exist

Similar issue happens if I try to pass any of the options, that are not defined in highs --help

I look forward to guidance in this regard. Thank you.

**Comment by jajhall** [comment, marked as answer]
Only a few of the options can be set directly as command-line options, see

https://ergo-code.github.io/HiGHS/stable/executable/

for `log_to_console` and other options, you need to create a file (say `foo.set`) with data such as

> log_to_console = false

and call HiGHS with the command line option

--options_file=foo.set

**Reply by phoenix665** [reply]
Thank you so much for your guidance.

I have another question, for some model size I am testing, the solution comes out to be "Infeasible" with default presolve settings. But the moment I turn presolve=off, I start getting some value on objective bounds. I am not quite able to place what might be the issue. If you can guide me about it?

**Reply by jajhall** [reply]
There are bugs in the MIP presolve that can lead to incorrect deduction of infeasibility. If your problems trigger one of these bugs, I can only suggest that you turn presolve=off. 

We are working on fixing these bugs, but it is far from easy to do

**Reply by phoenix665** [reply]
Yes. Thank you. I will work with presolve off.
